include/GafferSceneUI/SceneGadget.h

```diff
@@ -162,8 +164,13 @@ class GAFFERSCENEUI_API SceneGadget : public GafferUI::Gadget
 		) const;
 
 		/// Returns the bounding box of all the selected objects.
+		/// TODO - should this be deprecated in favour of queryBound?
 		Imath::Box3f selectionBound() const;
 
+		/// Queries the bound with additional parameters - "selected" queries only selected objects,
+		/// and "omitted" is a PathMatcher with paths to specifically omit
+		Imath::Box3f queryBound( bool selected = false, const IECore::PathMatcherData *omitted = nullptr ) const;
```

Comment:
Consolidating these seems reasonable, but there's also the original `Gadget::bound()` method to consider. With that in mind, `queryBound()` seems an odd choice of name. Maybe just a `bound( bool selectionOnly, const PathMatcher *omitted = nullptr )` overload? I think taking `PathMatcher *` rather than requiring the caller to wrap in `PathMatcherData` is preferable too - it's really just an implementation detail that it needs to be wrapped for the Renderer.

Reply:
Addressed.